### PR TITLE
Quizzespage players

### PR DESCRIPTION
**Pull request by kelseyedmodo**

### CODE REVIEW

**Comment by kelseyedmodo** [comment]
I don't really know how you want it sent back to you.. so I just fill up the page with the dataz  @blakeparkinson

File: public/javascripts/global.js [hunk 1]
```diff
@@ -129,14 +129,15 @@ function fetchQuiz(event) {
     var target = $(event.target);
     if (target.hasClass('quiz-btn')){
       var team = $('.selected-team'),
-          id = team.data('id');
+          id = team.data('team-id'),
```
Comment (blakeparkinson) [inline public/javascripts/global.js]: I forgot how this works. Is this the RB id or the API id? Don't we need both of them?

Comment (kelseyedmodo) [inline public/javascripts/global.js]: Right now I'm just pulling the API id through bc I haven't seen an immediate use for the RB id and that would make the url really long to pull both.

File: routes/teams.js [hunk 2]
```diff
@@ -27,8 +27,9 @@ router.get('/', function(res, res) {
 
 // when quiz endpoint is hit, insert a new quiz into mongo and return quiz_id
 router.get('/quiz', function(req, res) {
-  id = req.query["id"];   //this is the rosterblitz id, not the API one
-  quiz = createQuiz(id, res, returnItem);
+  id = req.query["team_id"];   //this is the API id
```
Comment (blakeparkinson) [inline routes/teams.js]: can we standardize these to ```req.query.team_id; req.query.league ``` etc...?

**Comment by blakeparkinson** [comment]
simple/beautiful:

image

**Comment by kelseyedmodo** [comment]
yea yea yea- I told you I was just barfing it out onto the page.

On Mon, Feb 9, 2015 at 10:36 PM, Blake Parkinson <notifications@github.com>
wrote:

> simple/beautiful:
>
> [image: image]
> <https://cloud.githubusercontent.com/assets/1864277/6122680/040f22aa-b0ac-11e4-9dcf-b9252095c547.png>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/blakeparkinson/sports2/pull/82#issuecomment-73651657>.
>



-- 
*Kelsey Gross*
Edmodo Customer Success
https://www.edmodo.com/kelseygross

File: public/javascripts/global.js (hunk 1 above)
Comment (blakeparkinson) [inline public/javascripts/global.js]: I think that makes sense for now. I'm still leery of being tied to the API ID. If a record of all of the teams is in mongo (which they should be, no?) wouldn't it be better to key off of the RB id, and then fetch an update from the API depending on our rules?

Comment (kelseyedmodo) [inline public/javascripts/global.js]: Well, probably better to key off the RB id.. but if we do that, I have to
carry the API id with it everywhere bc I need that to make any updates.

If we switch to a different API, I'll have to tweak a bunch of things
around anyways so I'm not that concerned about this for now.

On Mon, Feb 9, 2015 at 10:41 PM, Blake Parkinson <notifications@github.com>
wrote:

> In public/javascripts/global.js
> <https://github.com/blakeparkinson/sports2/pull/82#discussion_r24391886>:
>
> > @@ -129,14 +129,15 @@ function fetchQuiz(event) {
> >      var target = $(event.target);
> >      if (target.hasClass('quiz-btn')){
> >        var team = $('.selected-team'),
> > -          id = team.data('id');
> > +          id = team.data('team-id'),
>
> I think that makes sense for now. I'm still leery of being tied to the API
> ID. If a record of all of the teams is in mongo (which they should be, no?)
> wouldn't it be better to key off of the RB id, and then fetch an update
> from the API depending on our rules?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/blakeparkinson/sports2/pull/82/files#r24391886>.
>



-- 
*Kelsey Gross*
Edmodo Customer Success
https://www.edmodo.com/kelseygross

Comment (blakeparkinson) [inline public/javascripts/global.js]: Cool. Merging...

**Pull request merged**